src/__test__/components/plots/ContinuousEmbeddingPlot.test.jsx

```diff
@@ -6,69 +6,58 @@ import thunk from 'redux-thunk';
 import { Provider } from 'react-redux';
 import { Vega } from 'react-vega';
 
-import { Skeleton } from 'antd';
+import { ClipLoader } from 'react-spinners';
 import ContinuousEmbeddingPlot from '../../../components/plots/ContinuousEmbeddingPlot';
 import { initialEmbeddingState } from '../../../redux/reducers/embeddings/initialState';
 import initialCellSetsState from '../../../redux/reducers/cellSets/initialState';
-import initialGeneExpressionState, { initialExpressionState } from '../../../redux/reducers/genes/initialState';
+import initialGeneExpressionState from '../../../redux/reducers/genes/initialState';
 import initialExperimentState from '../../experimentSettings.mock';
 import { initialPlotConfigStates } from '../../../redux/reducers/componentConfig/initialState';
 
 jest.mock('localforage');
+
+const mockCellSets = {
```

Comment:
This is confusing because it defines a wrong structure of a `cellSet`, with `cellIds` specified as strings. It may be convenient to import the already existing https://github.com/biomage-ltd/ui/blob/master/src/__test__/cellSets.mock.js

Reply:
changed, as suggested, now imported from `cellSets.mock.js`